packages/js/product-editor/src/blocks/generic/number/README.md

```diff
@@ -79,6 +79,14 @@ The maximum numeric value that can be entered in the field.
 
 The amount that is incremented or decremented when using the up/down arrows.
 
+### showStepButtons
+
+- **Type:** `Boolean`
+- **Required:** `No`
+- **Default:** `false`
+
+Shows + and - buttons to increment/decrement the numeric value.
+
```

Comment:
I wonder if we even need this attribute, or at the very least, whether it should be `true` by default.

The underlying `input` element leaves the display of step buttons up to the browser when `type="number"` (which we aren't currently setting... see below) and and `step` is set.

Reply:
I'd suggest we have our step buttons work like the native browser ones, in that you don't decide whether to show them or not. They just show if the input is a number.

I think this would be more future-proof, because if we switch to native step controls in the future, we don't have any control whether they appear or not.